.circleci/config.yml

```diff
@@ -19,9 +16,75 @@ jobs:
       - run:
           name: Run unit tests
           command: |
-            export PYTHONPATH=$SMAUG_HOME/build:$PYTHONPATH
+            export PYTHONPATH=$SMAUG_HOME:$PYTHONPATH
             make test-run
+      - run:
+          name: Build documentation
+          command: |
+            apt install -y doxygen
+            doxygen Doxyfile
       - run:
           name: Download latest gem5-aladdin binary
           command:
             python .circleci/download_artifacts.py --api_token=${GEM5_ALADDIN_BUILD_ARTIFACT_TOKEN} --project=gem5-aladdin --artifact_name=gem5.opt --user=${USER} --download_loc=/tmp --filter=${BUILD_ARTIFACT_FILTER} --branch=${BUILD_ARTIFACT_BRANCH}
+      - persist_to_workspace:
+          root: docs
+          paths:
+            - html
+
+  docs_deploy:
+    docker:
+      - image: cimg/base:2020.01
+    steps:
+      - run:
+          name: Stop if doc deployment is disabled
+          command: |
+            if [[ ${ENABLE_DOC_DEPLOY} ]]; then
```

Comment:
By default, `ENABLE_DOC_DEPLOY` is not added in the user's own CI project, so no docs deployment will be enabled?

Reply:
Correct.